### PR TITLE
MNT Use shared travis config, use sminnee/phpunit

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,44 +1,4 @@
...
-    - $HOME/.composer/cache/files
-
-matrix:
-  include:
-    - php: 5.6
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
-    - php: 7.1
-      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_COVERAGE_TEST=1 PDO=1
-    - php: 7.2
-      env: DB=PGSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1
```
Comment (dnsl48) [inline .travis.yml:21]: We may need to add `PGSQL` to `cwp-jobs-range` to not reduce the coverage here

**Review by dnsl48 — CHANGES_REQUESTED** [review]
Either add PGSQL to `cwp-jobs-range`, or add another jobs variation (`cwp-jobs-range-pg`?)

File: .travis.yml (hunk 1 above)
Comment (emteknetnz) [reply .travis.yml:21]: You can use hybrid sessions on cloud as well, so make sense to use use 'standard' rather than 'cwp' for this instead

Comment (emteknetnz) [reply .travis.yml:21]: this is green now

**Review by dnsl48 — APPROVED** [review]

**Pull request merged**